ocaml/idl/datamodel.ml

```diff
@@ -7107,6 +7107,17 @@ let pool_remove_from_guest_agent_config = call
     ~allowed_roles:_R_POOL_ADMIN
     ()
 
+let pool_set_default_sr = call
```

Comment:
For consistency with the field name, it would be better to capitalise "SR" in `pool_set_default_SR`.